sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -203,6 +219,13 @@ public EncryptedBlobAsyncClient buildEncryptedBlobAsyncClient() {
         Objects.requireNonNull(blobName, "'blobName' cannot be null.");
         checkValidEncryptionParameters();
 
+        this.encryptionVersion = encryptionVersion == null ? EncryptionVersion.V1 : encryptionVersion;
+        if (EncryptionVersion.V1.equals(this.encryptionVersion)) {
+            LOGGER.warning("Client is being configured to use v1 of client side encryption, which is no longer "
```

Comment:
Similar comment about being logged at a lower level after the first time being logged.

Reply:
Added to list